docs/introduction/about.md

```diff
@@ -5,66 +5,58 @@ sidebar_label: What is Calimero?
 
-- Permissioned Private Calimero Shard owned by a consortium
+Calimero's private shards cater to various user groups:
 
-- Public Calimero Shard owned by a community
+- **Traditional (web2) businesses**: Calimero provides an easy-to-use dashboard that serves as a gateway for web2 businesses to enter the web3 world. It offers a convenient way to manage private shards and access the benefits of blockchain technology.
 
-Calimero shards can support up to 200k transactions per second, but a standard one shard deployment has the throughput of 5k transactions per second.
+- **Existing web3 businesses**: Businesses already operating in the web3 space can leverage Calimero's zero gas scaling and privacy features to enhance their operations.
 
-## Features
 
-- Interoperate with open source protocols and applications to enhance your application, from DeFi to NFTs.
+## Use Cases
 
-- [Swap your assets](https://docs.calimero.network/bridge/bridging/prerequisites) between private chains and other public blockchains seamlessly. 
+- **Interoperability with open-source protocols and applications**: Calimero allows you to enhance your application by interoperating with a wide range of protocols and applications, spanning from decentralized finance (DeFi) to non-fungible tokens (NFTs).
 
-- With Calimero, you can perform [cross-shard](https://docs.calimero.network/bridge/bridging/cross_shard_calls) transactions, which allow you to interact with other third parties on the public chain or other permissioned enterprise shards. 
+- **Seamless asset swapping**: You can effortlessly [wap assets](https://docs.calimero.network/bridge/bridging/prerequisites) between private chains and other public blockchains, ensuring fluidity and flexibility in managing your digital assets.
 
-- With the [Calimero SDKs](https://docs.calimero.network/calimero_sdk/installation), you can make smart contracts from private shards can call into public blockchain to interact with those applications.
+- **Cross-shard transactions**: Calimero enables [cross-shard](https://docs.calimero.network/bridge/bridging/cross_shard_calls)  transactions, empowering you to interact with third parties on the public chain or other permissioned enterprise shards.
 
-- Public smart contracts if setup can also call into private shard’s contracts (extended trust model, similar to Oracles).
+- **Interacting with public blockchain applications**: Leveraging [Calimero SDKs](https://docs.calimero.network/calimero_sdk/installation), smart contracts deployed on private shards can interact with public blockchains (extended trust model, similar to Oracles), expanding the reach and capabilities of your application.
 
-- An encrypted channel can be used for private shards to interact with each other. 
+- **Encrypted channel for private shard interaction**: Calimero supports secure and encrypted channels for private shards to interact with each other, safeguarding sensitive data and ensuring privacy.
 
```

Comment:
**Encrypted channels**: Calimero supports secure and encrypted channels that enable private shards to interact with each other, safeguarding sensitive data and ensuring privacy.